lib/context.js

```diff
@@ -1,13 +1,17 @@
 import { readFile, writeFile } from 'mz/fs'
 import { resolve } from 'path'
 import { homedir } from 'os'
+import { sync } from 'find-up'
 
 import { proxyStringToObject } from './utils/proxy'
 
 let context
 
 export function getConfigPath () {
-  return resolve(homedir(), '.contentfulrc.json')
+  const contentfulrc = '.contentfulrc.json'
+  const configPath = sync(contentfulrc)
+  if (!configPath) return resolve(homedir(), contentfulrc)
```

Comment:
Let's add some curly brackets here {}

While it might be okay here I prefer to have them. It is more clear to read and later changes won't cause that big git diffs.

Pure coding style pickiness here 🙈

Reply:
Using async would even force you to do it in a try-catch block which would then resolve this issue too.